src/engine/specification.md

```diff
@@ -119,12 +82,42 @@ This structure maps on the [`ExecutionPayload`](https://github.com/ethereum/cons
 1. `Object` - The state of the fork choice:
 - `headBlockHash`: `DATA`, 32 Bytes - block hash of the head of the canonical chain
 - `finalizedBlockHash`: `DATA`, 32 Bytes - block hash of the most recent finalized block
+- `payloadAttributes`: `Object|None` - instance of [`PayloadAttributes`](#PayloadAttributes) or `None`
 
 #### Returns
-None or an error
+
+`Object|Error` - Either instance of response object or an error. Response object:
+1. `status`: `String` - The result of updating the fork choice
+  - `SUCCESS` - The fork choice is successfully updated, and if requested, the payload build process has successfully started
+  - `SYNCING` - Sync process is in progress
 
 #### Specification
 
 1. This method call maps on the `POS_FORKCHOICE_UPDATED` event of [EIP-3675](https://eips.ethereum.org/EIPS/eip-3675#specification) and **MUST** be processed according to the specification defined in the EIP.
 
-2. Client software **MUST** respond with `4: Unknown block` error if the payload identified by either the `headBlockHash` or the `finalizedBlockHash` is unknown.
+2. Client software **MUST** return `SYNCING` status if the payload identified by either the `headBlockHash` or the `finalizedBlockHash` is unknown.
```

Comment:
It should probably add ", or while the sync process is in progress" as payloads might already been known in some sense but the sync yet not done